src/sentry/static/sentry/app/views/settings/project/projectTeams.jsx

```diff
@@ -108,6 +29,24 @@ class ProjectTeams extends AsyncView {
     );
   };
 
+  handleRemove = teamSlug => {
+    if (this.state.loading) {
+      return;
+    }
+
+    const {orgId, projectId} = this.props.params;
+    const team = this.state.allTeams.find(tm => tm.slug === teamSlug);
+
+    removeTeamFromProject(this.api, orgId, projectId, team.slug)
+      .then(() => this.handleRemovedTeam(team))
+      .catch(() => {
+        this.setState({
+          error: true,
```

Comment:
Do we want to put the entire view in an error state in this case? Would an error toast (`addErrorMessage`) be better?